modules/openapi-generator/src/main/resources/python/model_generic.mustache

```diff
@@ -12,7 +12,7 @@ import json
 
 class {{classname}}({{#parent}}{{{.}}}{{/parent}}{{^parent}}BaseModel{{/parent}}):
     """
-    {{{description}}}{{^description}}{{{classname}}}{{/description}}
+    {{#description}}{{{description}}}  # noqa: E501{{/description}}{{^description}}{{{classname}}}{{/description}}
```

Comment:
It's possible that this could be avoided if there were a nice way to convert newlines in the description text into something that mustache won't swallow up, but that's going to be hard to do generally.